### PR TITLE
feat: demo flow

### PR DESCRIPTION
**Pull request by ryanseee**
https://www.pivotaltracker.com/epic/show/4763118

### CODE REVIEW

**Comment by cavacado** [comment]
testing things out only right?
put to draft if thats the ccase

**Comment by ryanseee** [comment]
@cavacado ok i just converted it to draft, thanks zl!

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "ethers": "^5.4.3",
     "history": "^4.10.1",
     "lodash": "^4.17.21",
+    "magic-sdk": "^5.0.0",
```
Comment (simboonlong) [inline package.json]: looks like latest is 6ish already

File: src/common/contexts/helpers.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Magic } from "magic-sdk";
+import { MAGIC_API_KEY } from "../../config/";
+
+export const magic = new Magic(MAGIC_API_KEY, {
+  network: "ropsten", // fix to ropsten network only
+  // testMode: true,
+});
```
Comment (simboonlong) [inline src/common/contexts/helpers.tsx]: can remove comment

File: src/common/contexts/provider.tsx [hunk 3]
```diff
@@ -2,25 +2,34 @@ import { MessageTitle } from "@govtechsg/tradetrust-ui-components";
 import { ethers, providers, Signer } from "ethers";
 import React, { createContext, FunctionComponent, useContext, useEffect, useState } from "react";
 import { INFURA_API_KEY, NETWORK_NAME } from "../../config";
-import { utils } from "@govtechsg/oa-verify";
+import { magic } from "./helpers";
+
+export enum SIGNER_TYPE {
+  IDENTITY = "Identity",
+  METAMASK = "Metamask",
```
Comment (simboonlong) [inline src/common/contexts/provider.tsx:10]: `IDENTITY` is for what case ah?

File: src/common/contexts/provider.tsx [hunk 4]
```diff
@@ -2,25 +2,34 @@ import { MessageTitle } from "@govtechsg/tradetrust-ui-components";
...
+  MAGIC = "Magic",
+}
 
 const getProvider =
   NETWORK_NAME === "local"
     ? new providers.JsonRpcProvider()
-    : utils.generateProvider({ network: NETWORK_NAME, providerType: "infura", apiKey: INFURA_API_KEY });
+    : new ethers.providers.InfuraProvider(NETWORK_NAME, INFURA_API_KEY);
```
Comment (simboonlong) [inline src/common/contexts/provider.tsx]: no should use `generateProvider`, context -> https://github.com/Open-Attestation/adr/blob/master/provider.md

File: src/components/HomePageContent/HowItWorksSection/index.tsx [hunk 5]
```diff
@@ -83,6 +83,13 @@ export const HowItWorksSection: FunctionComponent = () => {
           >
             <h3 className="font-normal text-2xl">Get in Touch Now</h3>
           </Link>
+          {/* <Link
+            to="/demo"
+            className="px-4 py-2 mx-auto -mt-4 rounded-xl text-white bg-tangerine hover:bg-tangerine-600 hover:text-gray-200"
+            data-testid="demo"
+          >
+            <h3 className="font-normal text-2xl">Try our demo now</h3>
+          </Link> */}
```
Comment (simboonlong) [inline src/components/HomePageContent/HowItWorksSection/index.tsx]: remove for now

Comment (ryanseee) [reply src/components/HomePageContent/HowItWorksSection/index.tsx]: done.

File: src/common/contexts/provider.tsx (hunk 4 above)
Comment (ryanseee) [reply src/common/contexts/provider.tsx]: ok changed back to generateProvider()

File: src/common/contexts/helpers.tsx (hunk 2 above)
Comment (ryanseee) [reply src/common/contexts/helpers.tsx]: ok done.

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**